### ISSUE TITLE
Substantial Sample Application

### ISSUE THREAD

**Issue by reem** [iron/iron#150]
Should include widespread use of many middleware, API integration, database integration, etc. I'd love to see what a full app written in Iron actually looks like and it would be extremely helpful for people trying to learn the framework.

**Comment by untitaker** [comment]
I've written [mysteryshack](https://github.com/untitaker/mysteryshack), a small server implementing [remoteStorage](http://remotestorage.io/). Rough scope:

* Basic user and session management
* JSON API + very small subset of OAuth
* Administration interface for OAuth tokens

As an intermediate step we could clone http://flask.pocoo.org/community/poweredby/ using a wiki or doc page.

**Comment by michaelsproul** [comment]
Yeah, I like the idea of linking a few examoles, with a list of features used in each one. My current project is 'festivus' - a Postgres backed time-series database for power and energy data.

https://github.com/michaelsproul/festivus
